.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,36 @@
+#Adopted from https://gist.github.com/filipkis/fab89a09f54ffdd8e70c063755e7ce7f
+
+name: Update vendor/cache After Dependabot
+
+on:
+  push:
+    branches:
+      - 'dependabot/bundler/**'
+    paths:
+      - 'Gemfile.lock'
+
+jobs:
+  update-gems:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Ruby 2.7
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.7.1
+      - name: Installing additional dependencies
+        run: |
+          sudo apt-get install postgresql-client libpq-dev
+      - name: Bundle install
+        run: |
+          gem install bundler --no-rdoc --no-ri
+          bundle install
```

Comment:
do we need to run `bundle config set cache_all true` before this or is it the default when bundler is installed?

Reply:
Good catch, added `cache_all` and `cache_all_platforms`.